docs/language/ql-handbook/expressions.rst

```diff
@@ -279,6 +279,22 @@ The following aggregates are available in QL:
   evaluates to the empty set (instead of defaulting to ``0`` or the empty string).
   This is useful if you're only interested in results where the aggregation body is non-trivial.
 
+.. index: unique
+
+- ``unique``: This aggregate depends on the values of ``<expression>`` over all possible assignments to
+  the aggregation variables. If there is a unique value of the ``expression`` over the aggregation variables,
+  then the aggregate evaluates to that value.
+  Otherwise, the aggregate has no value.
+
+  For example, the following aggregation yields the positive integers ``1``, ``2``, ``3``, ``4``, ``5``.
+  For negative integers ``x``, the expressions ``x`` and ``x.abs()`` have different values and hence the
+  value for ``y`` in the aggregate expression is not uniquely determined. ::
+
+      from int x
+      where x in [-5 .. 5] and x != 0
+      select unique(int y | y = x and y = x.abs() | y)
```

Comment:
```suggestion
      select unique(int y | y = x or y = x.abs() | y)
```

If `and` is used here, then the removal of non-unique `y` happens in the formula rather than the aggregation function, right?

Reply:
Very true!